prettyprinter/test/Testsuite/Main.hs

```diff
@@ -138,7 +142,7 @@ content = frequency
     [ (1, pure emptyDoc)
     , (10, do word <- choose (minBound, maxBound :: Word8)
               let pgp8Word = toText (BSL.singleton word)
-              pure (pretty pgp8Word) )
+              pure (pretty (read (show pgp8Word) :: T.Text)) )
```

Comment:
I don't understand why the `read . show` bit is necessary. Could it be simplified?!

Reply:
`toText` above comes from `pgp-wordlist` package, so `pgp8Word` is a genuine `Text`, irrespective of `prettyprinter` flags. If `prettyprinter` is built with `text`, then `read . show` is just `id` here, but if `text` is disabled, `read . show` converts a genuine `Text` to a fake one. This is the simplest workaround, avoiding CPP.

Reply:
Oof, somehow this makes my head spin. I think I might prefer the CPP. Or define `textToText = read . show` and add an explanatory comment…